src/packages/controller/utils/find-one.js

```diff
@@ -25,7 +25,7 @@ export default async function findOne({
     include
   } = paramsToQuery(model, merge(defaultParams, params));
 
-  return await model.find(id)
```

Comment:
The `Query` interface is a lazy evaluated function that "becomes" a `Promise` when `.then` is called or in this case, is on the right hand side of an `await` expression. We should not trigger a `.then` call prematurely to allow for chaining in a `Controller` if necessary.